### DISCUSSION TITLE
Issue in setting-up pouchdb in my Vite-react application.

### DISCUSSION THREAD

**Discussion by rajarshigit14** [pouchdb/pouchdb#8879] [Q&A] [answered]
My objective is to make the application work offline Vite-PWA-Plugin and PouchDB. I have installed pouchdb-browser, pouchdb-adapter-idb but whatever i do it is giving me this error : 
pouchdb.js?v=fb35a119:31 Uncaught ReferenceError: global is not defined
    at node_modules/immediate/lib/mutation.js (pouchdb.js?v=fb35a119:31:20)
    at __require (chunk-ZS7NZCD4.js?v=fb35a119:8:50)
    at node_modules/immediate/lib/index.js (pouchdb.js?v=fb35a119:114:7)
    at __require (chunk-ZS7NZCD4.js?v=fb35a119:8:50)
    at pouchdb.js?v=fb35a119:817:32
node_modules/immediate/lib/mutation.js @ pouchdb.js?v=fb35a119:31
__require @ chunk-ZS7NZCD4.js?v=fb35a119:8
node_modules/immediate/lib/index.js @ pouchdb.js?v=fb35a119:114
__require @ chunk-ZS7NZCD4.js?v=fb35a119:8
(anonymous) @ pouchdb.js?v=fb35a119:817

Can someone help me out with this issue?

**Comment by SourceR85** [comment, marked as answer]
Duplicate of #8516
Please wait for the next release, #8849 already fixed this issue.
